react/components/CodeBlock.tsx

```diff
@@ -14,9 +14,12 @@ const CodeBlock: FC<Props> = ({ language, value }) => {
     }
   }, [codeBlockRef])
 
+  // highlight.js does not support `.jsonc` files
+  const codeBlockLanguage = language === 'jsonc' ? 'json' : language
```

Comment:
Very nice! Would it be great to default to `json` if it doesn't have a language at all?

Reply:
I thought about it some time ago! 
I think every snippet in Markdown files should always include the language. If we set this default to JSON, people would start to rely on that, and we would be encouraging them to write bad Markdown.